packages/core/src/Ignition.ts

```diff
@@ -191,7 +193,48 @@ export class Ignition {
 
     const { executionGraph } = transformResult;
 
-    return { deploymentGraph, executionGraph };
+    return {
+      deploymentGraph,
+      executionGraph,
+      networkName: networkNameByChainId[chainId] ?? "unknown",
+    };
+  }
+
+  public async info(moduleName: string): Promise<Deployment[]> {
+    log(`Start info`);
+
+    const journalData: {
+      [k: string]: Array<DeployStateExecutionCommand & { chainId: number }>;
+    } = {};
+    for await (const command of this._journal.readAll()) {
+      const network = networkNameByChainId[command.chainId] ?? "unknown";
+
+      if (journalData[network] === undefined) {
+        journalData[network] = [];
+      }
+
+      journalData[network].push(command);
```

Comment:
Can we restructure the journalData to be id'ed by chainId. I realise this makes the value need to be a more complex object than an array. It has an advantage though in that two different `chainId`s that are not in the network map, will be kept separate but labelled 'unknown' rather than both being merged into 'unknown'.

Reply:
oh good catch